modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -161,6 +182,89 @@ public int call(final String[] args, final Cfg cfg) {
                         }
                     }
                     break;
+
+                // Determines network config folder path
+                case "-n":
+                case "--network":
```

Comment:
Currently using the correct number of args but an incorrect arg -> `./aion.sh -c main` tells me that the network is invalid but then it also tries to start the kernel up and crashes after that. Should exit before starting the kernel.

Reply:
Same as below :)